java/src/test/java/ai/rapids/cudf/ColumnViewNonEmptyNullsTest.java

```diff
@@ -41,11 +42,10 @@ void testAndNullReconfigureNulls() {
          ColumnVector intResult = v1.mergeAndSetValidity(BinaryOp.BITWISE_AND, v0);
          ColumnVector v2 = ColumnVector.fromStrings("0", "100", "1", "2", "MIN_VALUE", "3");
          ColumnVector stringResult = v2.mergeAndSetValidity(BinaryOp.BITWISE_AND, v0, v1);
```

Comment:
nit: Can we get some documentation updates about what BITWISE_AND and BITWISE_OR do?  It is not really intuitive because we often thing about this in terms of a null mask, but it is a validity mask, so BITWISE_AND is the same as

```
try (ColumnVector shouldOutputNotBeNull = columns[0].isNotNull()) {
  for (int i = 1; i < columns.length; i++) {
    try (ColumnVector tmpIsNotNull = column[i].isNotNull()) {
      ColumnVector tmpCombined = tmpIdNotNull.and(shouldOutputNotBeNull)
      shouldOutputNotBeNull.close()
      shouldOutputNotBeNull = tmpCombined
    }
  }
  return shouldOutputNotBeNull.ifElse(this, null_scalar)
}
```

That might be a bit long though. If you feel it is too long feel free to ignore this. Or if you have a better way to describe it that works too.

Perhaps just something where we change it from null masks to validity buffers in the comments.

```
 /**
   * Create a deep copy of the column while replacing the validity buffers. The resultant validity buffer is the
   * bitwise merge of validity in the columns given as arguments. Please note that validity is the opposite of 
   * a null.  So BITWISE_AND will put a null in the output if either input values is a null, and BITWISE_OR
   * will put a null value in the output only if all input values are null.
   * The result will be sanitized to not contain any non-empty nulls in case of nested types
   *
```